datasync/README.md

```diff
@@ -24,7 +24,7 @@
 
 You may be asked to log in to Google Drive, if this is the first time you're running the script or if it's been a while.
 
-If you get the error `google.auth.exceptions.RefreshError: ('invalid_grant: Bad Request', {'error': 'invalid_grant', 'error_description': 'Bad Request'})` try deleting `tokens/token.json` to force it to redo the Google auth.
+If you get the error `google.auth.exceptions.RefreshError: ('invalid_grant: Bad Request', {'error': 'invalid_grant', 'error_description': 'Bad Request'})`  or `google.auth.exceptions.RefreshError: ('invalid_grant: Token has been expired or revoked.', {'error': 'invalid_grant', 'error_description': 'Token has been expired or revoked.'})` try deleting `tokens/token.json` to force it to redo the Google auth.
```

Comment:
I don't like this bodge, so I filed https://github.com/eldang/ballard-4culture/issues/12 for a better fix.  But that's my lowest priority on this project right now because the workaround is still usable.

Reply:
Yes that seems reasonable. Also I now know what 'bodge' means :)

Reply:
I will never pass up a chance for some cultural education!